release_notes/ocp_3_4_release_notes.adoc

```diff
@@ -1702,3 +1702,194 @@ openshift3/registry-console:3.4-30
+ timeout for the image pull was increased. The diagnostics now run in slow
+ environments.
+ (link:https://bugzilla.redhat.com/show_bug.cgi?id=1481550[*BZ#1481550*])
+
+* The OpenShift node proxy previously did not support using a specified IP
+address. This could prevent correct operation on hosts with multiple network
+interface cards. The OpenShift node process already accepts a
+`--bind-address=<ip address>:<port>` command-line flag and `bindAddress:`` configuration
+file option for the multiple network interface card case. The proxy
+functionality has been fixed to respect these options. When `--bind-address` or
+`bindAddress` are used, the OpenShift node proxy should work correctly when the
+OpenShift node host has multiple network interface cards.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1489023[*BZ#1489023*],
+link:https://bugzilla.redhat.com/show_bug.cgi?id=1489024[*BZ#1489024*])
+
+* Iptables called too often and unnecessarily. Therefore, time-outs would wait for
+iptables operations to finish. This bug fix changes the code so that it skips
+reloads when the iptables rules are unchanged. There are now fewer calls to
+iptables and, therefore, less time-outs.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1501517[*BZ#1501517*])
+
+[discrete]
+===== Pod
+
+* There was a symbolic link error for the log file of every pod started when the
+docker log driver was journald. Log symlink creation that fails when using
+journald logging driver was skipped. This bug fix resolves the issue.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1434942[*BZ#1434942*])
+
+* Currently, pod anti-affinity is respected across projects. Pod A from Project 1
+will not land on node where Pod B from Project 2 is running, if pod
+anti-affinity is enabled when scheduling Pod A. While scheduling Pod A, check
+for pod anti-affinity only within the project of Pod A. Pod anti-affinity will
+not be respected across projects.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1492194[*BZ#1492194*])
+
+[discrete]
+===== Storage
+
+* The volumePath that included the datastore name was parsed incorrectly. The same
+applies to volumePath that included datacluster and datastore names. It is not
+possible to attach persistent volumes that have the above described volumePath
+values. volumePath is now parsed correctly. Persistent volumes that have the
+above described volumePath values are attached
+correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1497042[*BZ#1497042*])
+
+[discrete]
+===== Security
+
+* An attacker with knowledge of the given name used to authenticate and access
+Elasticsearch can later access it without the token, bypassing authentication.
+This attack also requires that the Elasticsearch be configured with an external
+route, and the data accessed is limited to the indices.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1501986[*BZ#1501986*])
+
+[[ocp-3-4-rhsa-2017-3389-images]]
+==== Images
```

Comment:
Not critical, but usually we have the image list right after the intro part (before the list of BZs) in this scenario, right?